Data/Scripts/System/Skills/Taming.cs

```diff
@@ -303,13 +303,19 @@ protected override void OnTick()
 					{
 						m_Tamer.RevealingAction();
 
-						switch ( Utility.Random( 5 ) )
+						switch ( Utility.Random( 9 ) )
```

Comment:
Random should be rolling 11 due to number of cases.

```suggestion
						switch ( Utility.Random( 11 ) )
```